cmd/sriovdp/sriov-device-plugin.go

```diff
@@ -515,6 +527,17 @@ func (sm *sriovManager) Allocate(ctx context.Context, rqt *pluginapi.AllocateReq
 		envmap["SRIOV-VF-PCI-ADDR"] = pciAddrs
 
 		containerResp.Envs = envmap
+
+		if isRdmaDevice(id){
```

Comment:
This will only configure the rdma device if the last device in ``Allocate`` request is rdma capable. perhaps we shall move this into the for loop against ``container.DevicesIDs`` above. Further improvement might be that the rdma could be a property of device that discovered and recorded in ``discoverNetworks``  function and we only need to check whether the device has rdma property in the ``Allocate`` call, but this requires changes in the main ``sriovManager`` data structure.